### PR TITLE
[FIX] val is null error during wizard view records fetch

### PR DESCRIPTION
**Pull request by hugho-ad**
Also fixed js error triggered because there is not tree view defiend

FIX https://git.vauxoo.com/vauxoo/instance/issues/529

Dummy 
https://git.vauxoo.com/vauxoo/instance/merge_requests/629

### CODE REVIEW

**Comment by hugho-ad** [comment]
@nhomar can you merge please

You can test it in http://runbot.vauxoo.com/runbot/build/50136

**Comment by luiseevaquer** [comment]
@nhomar 

Do I cancel the [PR](https://github.com/odoo/odoo/pull/22811) to odoo? 
because this solution works well

**Comment by hugho-ad** [comment]
@moylop260  can you merge this please

Regrds!

File: mass_editing/wizard/mass_editing_wizard.py [hunk 1]
```diff
@@ -268,4 +268,12 @@ def read(self, fields, load='_classic_read'):
         if fields:
             # We remove fields which are not in _fields
             real_fields = [x for x in fields if x in self._fields]
-        return super(MassEditingWizard, self).read(real_fields, load=load)
+        res = super(MassEditingWizard, self).read(real_fields, load=load)
+        # In order to avoid null value in client side we set dummy values fo
+        # dynamic fields build by fields_view_get
+        for item in res:
+            for field in fields:
```
Comment (moylop260) [inline mass_editing/wizard/mass_editing_wizard.py]: You can save a lot of time (performance) avoiding a loop-of-loop for a same result.
You can use `set` too.

 I mean,
Improvement 1.0 avoiding loop-of-loop
```python
...
res = super(MassEditingWizard, self).read(real_fields, load=load)
dummy_fields  = []
for field in fields:
    if field not in self._fields:
        dummy_fields.append(field)
dummy_fields_values = dict.fromkeys(dummy_fields)
for item in res:
    item.update(dummy_fields_values)
```


Improvement 2.0 using `set`
```python
...
res = super(MassEditingWizard, self).read(real_fields, load=load)
dummy_fields = dict.fromkeys(set(fields) - set(self._fields))
for item in res:
    item.update(dummy_fields)
```

The performance is so faster in the last one vs the current one and the code is so clear.

**Comment by hugho-ad** [comment]
@moylop260 

Thanks! I already apply the changes

Regards!

**Comment by moylop260** [comment]
Could you create the PR to OCA, please?

**Pull request merged**